frontend/packages/core-auth/src/components/AuthWrapper.tsx

```diff
@@ -98,17 +96,20 @@ const AuthWrapper: React.FC<AuthWrapperProps> = ({
         .then((resp) => {
           setTokenResponse(resp);
         })
-        .catch((err) => {
+        .catch(() => {
           setTokenResponse(undefined);
         });
     }
-  }, [tokenResponse?.shouldRefresh(), discovery]);
+  };
+  let refreshTokenIntervalId: NodeJS.Timeout | null = null;
```

Comment:
I think you probably want this to be a `ref` so that is doesn't get set to null on each render.

```
const refreshTokenIntervalId = React.useRef<NodeJS.Timeout | null>(null);
...
if (refreshTokenIntervalId.current != null) clearInterval(refreshTokenIntervalId.current);
refreshTokenIntervalId.current = setInterval(()=>refreshToken(), 5000);
```

Reply:
That's neat, I didn't know about `useReft`, thanks